app/controllers/haikus_controller.rb

```diff
@@ -13,20 +13,18 @@ def show
   def create
     haiku = Haiku.new(haiku_params)
     if haiku.save
-      render json: haiku
-      head 201
+      render json: haiku.lines
```

Comment:
Do we need a status 201 here ? I think by default the status code is going to be 200